multi_label_trainer.ipynb

```diff
@@ -0,0 +1,818 @@
+{
```

Comment:
<p>Before loading the data, once we should check if there are any redundant titles in the dataset or not. The search queries we used to scrape the data could infact overlap.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Or should we do that after loading the dataframe? </p><p> </p><p>If before, then I suggest it's done before we serialize the initial dataframe. </p>

Reply:
I thinkk we shoul do it before serialization.

Reply:
I'll update the data scraping notebook with the necessary pre-processing.

Reply:
<p>Alright. </p>